jazzmin/templates/admin/object_history.html

```diff
@@ -39,7 +39,7 @@ <h4 class="card-title">
                                     <i class="fas fa-{{ action_message|get_action_icon }} bg-{{ action_message|get_action_color }}"></i>
                                     <div class="timeline-item">
                                       <h3 class="timeline-header no-border">
-                                          <a href="{% url 'admin:auth_user_change' action.user.id %}" target="_blank">
+                                          <a href="{% jazzy_admin_url action.user %}" target="_blank">
```

Comment:
Probably you noticed already, but it should be:

```
{{ action.user|jazzy_admin_url }}
```

Reply:
dang its a filter

Reply:
Loving our test coverage right now